Bellerophon/BellerophonTests/BellerophonTests.swift

```diff
@@ -10,27 +10,158 @@ import XCTest
 @testable import Bellerophon
 
 class BellerophonTests: XCTestCase {
-    
+
+    enum ResponseCases: Int {
+        case KillSwitchOffForceUpdateOff = 0b00
+        case KillSwitchOffForceUpdateOn = 0b01
+        case KillSwitchOnForceUpdateOff = 0b10
+        case KillSwitchOnForceUpdateOn = 0b11
+    }
+
+    var responseArray: [BellerophonResponse]!
+
+    var shouldForceUpdateIsCalled: Bool!
+
+    var currentIdx: Int = 0
+
     override func setUp() {
         super.setUp()
-        // Put setup code here. This method is called before the invocation of each test method in the class.
+
+        responseArray = [BellerophonResponse]()
+
+        responseArray.append(BellerophonResponse(isAPIInactive: false, shouldForceUpdate: false,
+            interval: Double(ResponseCases.KillSwitchOffForceUpdateOff.rawValue), userMessageStr: "Both killSwitch and forceUpdate are turned off"))
+        responseArray.append(BellerophonResponse(isAPIInactive: false, shouldForceUpdate: true,
+            interval: Double(ResponseCases.KillSwitchOffForceUpdateOn.rawValue), userMessageStr: "Only forceUpdate is turned on"))
+        responseArray.append(BellerophonResponse(isAPIInactive: true, shouldForceUpdate: false,
+            interval: Double(ResponseCases.KillSwitchOnForceUpdateOff.rawValue), userMessageStr: "Only killSwitch is turned on"))
+        responseArray.append(BellerophonResponse(isAPIInactive: true, shouldForceUpdate: true,
+            interval: Double(ResponseCases.KillSwitchOnForceUpdateOn.rawValue), userMessageStr: "Both killSwitch and forceUpdate are turned on"))
+        
+        MockBPManager.mockSharedInstance.delegate = self
+        MockBPManager.mockSharedInstance.killSwitchView = UIView()
+
+        shouldForceUpdateIsCalled = false
+        
+        MockBPManager.mockSharedInstance.displayKillSwitchIsCalled = false
+        MockBPManager.mockSharedInstance.dismissKillSwitchIfNeededIsCalled = false
+        MockBPManager.mockSharedInstance.startAutoCheckingIsCalled = false
     }
     
     override func tearDown() {
-        // Put teardown code here. This method is called after the invocation of each test method in the class.
+        MockBPManager.mockSharedInstance.delegate = nil
+
         super.tearDown()
     }
-    
-    func testExample() {
-        // This is an example of a functional test case.
-        // Use XCTAssert and related functions to verify your tests produce the correct results.
+
+    func testResponseKillSwitchOffForceUpdateOff() {
```

Comment:
we should follow the new PiOS format for test functions

`test_responseKillSwitch_OffForceUpdateOff`